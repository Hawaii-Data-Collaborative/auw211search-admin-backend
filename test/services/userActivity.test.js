const { getUserActivityEventNames } = require('../../src/services/userActivity')

test('getUserActivityEventNames()', async () => {
  const result = await getUserActivityEventNames()
  expect(result).toMatchSnapshot()
})
