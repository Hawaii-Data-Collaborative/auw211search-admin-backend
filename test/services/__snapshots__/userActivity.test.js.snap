// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getUserActivityEventNames() 1`] = `
[
  "PageLoad",
  "Referral",
  "Referral.Phone",
  "Referral.Phone.Call",
  "Referral.Phone.CopyNumber",
  "Referral.Phone.ShowInfo",
  "Referral.Website",
  "Referral.Website.CopyWebsite",
  "Referral.Website.OpenInNewTab",
  "Referral.Website.ShowInfo",
  "Search",
  "Search.Category",
  "Search.Keyword",
  "Search.Taxonomy",
]
`;
